src/SDKs/_metadata/storagesync_resource-manager.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Please pull latest changes from remote psSdkJson6, run msbuild build.proj and regenerate the code